adflow/pyADflow.py

```diff
@@ -4516,259 +4482,244 @@ def setOption(self, name, value):
+            'timeAccuracy':[int, [2, 1, 3]],
+            'nTimeStepsCoarse':[int, 48],
+            'nTimeStepsFine':[int, 400],
+            'deltaT':[float, 0.010],
+            'useALE':[bool, True],
+            'useGridMotion':[bool, False],
+            'coupledSolution':[bool, False],
 
             # Time Spectral Parameters
-            'timeintervals': [int, 1],
-            'alphamode':[bool, False],
-            'betamode':[bool, False],
-            'machmode':[bool, False],
-            'pmode':[bool, False],
-            'qmode':[bool, False],
-            'rmode':[bool, False],
-            'altitudemode':[bool, False],
-            'windaxis':[bool, False],
-            'alphafollowing':[bool,True],
-            'tsstability': [bool, False],
+            'timeIntervals': [int, 1],
+            'alphaMode':[bool, False],
+            'betaMode':[bool, False],
+            'machMode':[bool, False],
+            'pMode':[bool, False],
+            'qMode':[bool, False],
+            'rMode':[bool, False],
+            'altitudeMode':[bool, False],
+            'windAxis':[bool, False],
+            'alphaFollowing':[bool,True],
+            'TSStability': [bool, False],
 
             # Convergence Parameters
-            'l2convergence':[float, 1e-6],
-            'l2convergencerel':[float, 1e-16],
-            'l2convergencecoarse':[float, 1e-2],
-            'maxl2deviationfactor':[float, 1.0],
+            'L2Convergence':[float, 1e-6],
+            'L2ConvergenceRel':[float, 1e-16],
+            'L2ConvergenceCoarse':[float, 1e-2],
+            'maxL2DeviationFactor':[float, 1.0],
 
             # Newton-Krylov Parameters
-            'usenksolver':[bool, False],
-            'nkswitchtol':[float, 2.5e-4],
-            'nksubspacesize':[int, 60],
-            'nklinearsolvetol':[float, 0.3],
-            'nkuseew':[bool, True],
-            'nkadpc':[bool, False],
-            'nkviscpc':[bool, False],
-            'nkasmoverlap':[int, 1],
-            'nkpcilufill':[int, 2],
-            'nkjacobianlag':[int, 20],
-            'applypcsubspacesize':[int, 10],
-            'nkinnerpreconits':[int, 1],
-            'nkouterpreconits':[int, 1],
-            'nkcfl0':[float, 100.0],
-            'nkls':[str, 'cubic'],
-            'nkfixedstep':[float, 0.25],
-            'rkreset':[bool, False],
-            'nrkreset':[int, 5],
+            'useNKSolver':[bool, False],
+            'NKSwitchTol':[float, 2.5e-4],
+            'NKSubspaceSize':[int, 60],
+            'NKLinearSolveTol':[float, 0.3],
+            'NKUseEW':[bool, True],
+            'NKADPC':[bool, False],
+            'NKViscPC':[bool, False],
+            'NKASMOverlap':[int, 1],
+            'NKPCILUFill':[int, 2],
+            'NKJacobianLag':[int, 20],
+            'applyPCSubspaceSize':[int, 10],
+            'NKInnerPreconIts':[int, 1],
+            'NKOuterPreconIts':[int, 1],
+            'NKLS':[str, ['cubic', 'none', 'non monotone']],
```

Comment:
similarly, non-monotone

Reply:
I was actually thinking of making this one word based on how prevalent 'nonmonotone line search' is compared to 'non-monotone line search'.

Reply:
Haha I find `nonmonotone` incredibly illegible, but that's still better than `non monotone` IMO.